### ISSUE TITLE
Error when push a FastjsDom into parent container

### ISSUE THREAD

**Issue by xiaodong2008** [fastjs-team/core#100]
### What is the current behavior?

```text
jsfast.js?v=e4690689:541 Uncaught TypeError: el.appendChild is not a function
    at _FastjsDom.push (jsfast.js?v=e4690689:541:22)
    at newStar (App.vue:44:12)
    at Proxy.mounted (App.vue:54:5)
    at chunk-PEY6ZZ6F.js?v=e4690689:4324:88
    at callWithErrorHandling (chunk-PEY6ZZ6F.js?v=e4690689:1659:19)
    at callWithAsyncErrorHandling (chunk-PEY6ZZ6F.js?v=e4690689:1666:17)
    at hook.__weh.hook.__weh (chunk-PEY6ZZ6F.js?v=e4690689:4304:19)
    at flushPostFlushCbs (chunk-PEY6ZZ6F.js?v=e4690689:1841:41)
    at render2 (chunk-PEY6ZZ6F.js?v=e4690689:8132:7)
    at mount (chunk-PEY6ZZ6F.js?v=e4690689:5375:13)
```

### What is the expected behavior?

It should push span into `.background`

### Steps to reproduce the problem

Code:
```javascript
dom
          .newEl("span", {
            class: "star",
            css: {
              left: rand(0, 100) + "vw",
              top: rand(0, 100) + "vh",
              transform: `scale(${rand(0, 150) / 100 + 0.5})`,
            },
          })
          .push(dom.select(".background")).el
          .then((star) => {
            star.addClass("show").then(() => {
              star.addClass("hide").then(() => {
                star.remove();
              }, 1000);
            }, rand(3000, 15000))
          }, 10)
          .then(newStar, rand(100, 2000))
```

### System Info

```Shell
Version: `v1.0.0-alpha.6`
```

### Confirmations

- [X] I have read the [Code of Conduct](https://docs.fastjs.dev/other/code-of-conduct.html)
- [X] I have read the [Issue Reporting Guidelines](https://github.com/fastjs-team/fastjs-next/blob/main/.github/contributing.md#issue-reporting)
- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate
- [X] I know that any inappropriate content may cause the maintainer to take action base on the [Code of Conduct](https://docs.fastjs.dev/other/code-of-conduct.html)

**Comment by xiaodong2008** [comment]
Reproduce: https://stackblitz.com/edit/fastjs-reproduce-jqpuxd?file=src%2Fmain.ts
It seems that it only throws an error when giving a FastjsDomList

**Comment by xiaodong2008** [comment]
Fixed: 41d8cfa82e272ba6f3deb0b4b79a51ffed1f6a96
